src/test/data/ManualTesting/[W09-B1][UniFy]TestScript.adoc

```diff
@@ -525,6 +527,13 @@ Remark: [Likes CS2103T] +
 Photo:  +
 Tags: [Legend][Myth][CS2103T][Man][Prof]
 
+*Type in:* ph 1 ph/[A path of txt file]+
```

Comment:
Be sure to update the list of commands executed in the History Command test below

Reply:
oh ok!